n_runs: 2
n_threads: 4
map_image: empty_4x4
buffer_length: 10000
batch_size: 1000
n_episodes: 10000
n_agents: 2
episode_length: 30
steps_per_update: 1
save_interval: 1000
eval_interval: 100
actor_hidden_dim: 64
critic_hidden_dim: 64
seed: 0
tau: 0.99
gamma: 0.95
auto_entropy: true
device: cpu
